.run/Python tests in tests.run.xml

```diff
@@ -0,0 +1,27 @@
+<component name="ProjectRunConfigurationManager">
```

Comment:
🤔 What is this file?

Reply:
Ah this is the pycharm run config, meant to add to gitignore